### ISSUE TITLE
Typescript definition problems

### ISSUE THREAD

**Issue by meriturva** [angular-ui/ui-router#2035]
Hi! we have few errors when we integrate the definition file angular-ui-router.d.ts on our projects.
Here the errors:

```
Import declaration conflicts with local declaration of 'ng'	
Cannot find name 'IServiceProvider'.
Cannot find name 'IPromise'.
```

I think namespaces are changed...my two cents.

**Comment by eddiemonge** [comment]
Um no idea. Maybe someone with TypeScript could help out here? My initial reaction is to drop that file altogether but thats because I don't use TypeScript yet.

**Comment by christopherthielen** [comment]
The ts file was user contributed. We added it to the repo as a courtesy. If there are errors, I'd love if you would submit a PR.

**Comment by licinioamendes** [comment]
Problem is that on `api/angular-ui-router.d.ts` module is declared as `declare module ng.ui` and must be `declare module angular.ui {`.

File seems a bit outdated I can take a look at it if needed.

**Comment by wesleycho** [comment]
This should be addressed with 63f4476 from #1990. Please use the implementation in https://github.com/borisyankov/definitelytyped
